homeassistant/components/velux/config_flow.py

```diff
@@ -1,16 +1,21 @@
 """Config flow for Velux integration."""
 
+from typing import Any
+
 from pyvlx import PyVLX, PyVLXException
 import voluptuous as vol
 
+from homeassistant.components.dhcp import DhcpServiceInfo
 from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
-from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_NAME, CONF_PASSWORD
 import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.device_registry import format_mac
 
 from .const import DOMAIN, LOGGER
 
 DATA_SCHEMA = vol.Schema(
     {
+        vol.Optional(CONF_NAME): cv.string,
```

Comment:
We don't allow a user settable name for integrations other than helpers. We should set the entry title to somethign the user can recognize

Reply:
OK, I removed the name.